test/suites/host-ip-change.robot

```diff
@@ -0,0 +1,233 @@
+*** Settings ***
```

Comment:
This file should move to a different directory so it is not run in the normal e2e job (I have no idea what will happen if we try to change the IP of an EC2 instance). If you put it in the suites-ostree directory and add a scenario, it can run in a VM in the metal job where changing the IP won't disrupt any of the other tests. See https://github.com/openshift/microshift/tree/main/test for details about how that works. Several of the other developers have added scenarios now and should be able to help you if you get stuck making your first one.

Reply:
@dhellmann It is in suites directory test/suites/host-ip-change.robot